### PR TITLE
Update misc tools

### PR DESCRIPTION
**Pull request by jacob-delgado**

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: docker/build-tools/Dockerfile [hunk 1]
```diff
@@ -42,7 +42,7 @@ ENV GO_JUNIT_REPORT_VERSION=df0ed838addb0fa189c4d76ad4657f6007a5811c
 ENV GOCOVMERGE_VERSION=b5bfa59ec0adc420475f97f89b58045c721d761c
 ENV GOIMPORTS_VERSION=v0.1.0
 ENV BENCHSTAT_VERSION=9c9101da8316
-ENV GH_VERSION=2.24.3
+ENV GH_VERSION=2.25.0
```
Comment (ericvn) [inline docker/build-tools/Dockerfile:45]: 2.25.1

**Review by ericvn — DISMISSED** [review]

**Comment by ericvn** [comment]
Hugo was updated in netlify with https://github.com/istio/istio.io/pull/12903

**Comment by ericvn** [comment]
I expect we want to hold off on merging this and updating build-tools until after the istio-private work has completed, likely April 5.

**Comment by ericvn** [comment]
Let's merge this and then when the Go updates are ready they can merge on top.

**Pull request merged**